support-models/src/main/scala/com/gu/support/zuora/api/SubscriptionData.scala

```diff
@@ -37,15 +38,22 @@ object RatePlanCharge {
       .add(endDateCondition, Json.fromString(subscriptionEnd)))
   implicit val contributionDecoder: Decoder[ContributionRatePlanCharge] = decapitalizingDecoder
 
+  implicit val introductoryPriceEncoder: Encoder[IntroductoryPriceRatePlanCharge] = capitalizingEncoder[IntroductoryPriceRatePlanCharge]
+    .mapJsonObject(_
```

Comment:
would it be clearer if we change this to `{ .. }` instead of `( ... )` and give the param a name e.g. 
```suggestion
    .mapJsonObject { somethingName =>
      somethingName.add(triggerEvent, Json.fromString(specificEvent))
    }
```

Reply:
could do, it would just be `mapJsonObject(jsonObject => jsonObject.add...` which doesn't really seem worth it to me as the method name already tells you what the type is.